opm/autodiff/FlowMainEbos.hpp

```diff
@@ -715,6 +710,48 @@ namespace Opm
         std::unordered_set<std::string> defunctWellNames() const
         { return ebosSimulator_->gridManager().defunctWellNames(); }
 
+        data::Solution computeLegacySimProps_()
+        {
+            const int* dims = UgGridHelpers::cartDims(grid());
+            const int globalSize = dims[0]*dims[1]*dims[2];
+
+            data::CellData tranx = {UnitSystem::measure::transmissibility, std::vector<double>( globalSize ), data::TargetType::INIT};
+            data::CellData trany = {UnitSystem::measure::transmissibility, std::vector<double>( globalSize ), data::TargetType::INIT};
+            data::CellData tranz = {UnitSystem::measure::transmissibility, std::vector<double>( globalSize ), data::TargetType::INIT};
+
+            const auto& eclTrans = ebosSimulator_->problem().eclTransmissibilities();
+            const auto& globalCell = grid().globalCell();
+            size_t num_faces = grid().numFaces();
+            auto fc = UgGridHelpers::faceCells(grid());
+            for (size_t i = 0; i < num_faces; ++i) {
+                auto c1 = std::min(fc(i,0), fc(i,1));
+                auto c2 = std::max(fc(i,0), fc(i,1));
+
+                if (c1 == -1 || c2 == -1)
+                    // boundary
+                    continue;
+
+                int gc1 = globalCell.size()?globalCell[c1]:c1;
+                int gc2 = globalCell.size()?globalCell[c2]:c2;
+
+                if (gc2 - gc1 == 1) {
+                    tranx.data[c1] = eclTrans.transmissibility(c1, c2);
+                }
+
+                if (gc2 - gc1 == dims[0]) {
+                    trany.data[c1] = eclTrans.transmissibility(c1, c2);
+                }
+
+                if (gc2 - gc1 == dims[0]*dims[1]) {
+                    tranz.data[c1] = eclTrans.transmissibility(c1, c2);
+                }
+            }
+            return
+              { {"TRANX" , tranx},
+                {"TRANY" , trany} ,
+                {"TRANZ" , tranz } };
+        }
+
```

Comment:
good suggestion. I'd like to do this but the problem is that `eclTrans` is not available in `DerivedGeology`.